### PR TITLE
feat: initial

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        ./.sonar/scanner/dotnet-sonarscanner begin /k:"Informatievlaanderen_basisregisters-docker-utilities" /o:"informatievlaanderen" /d:sonar.login="${{ secrets.VBR_SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io" /d:sonar.cs.dotcover.reportsPaths=dotCover.Output.html > /dev/null 2>&1
+
+    - name: Build
+      shell: bash
+      run: |
+        dotnet build --nologo --no-restore --no-incremental --configuration Debug basisregisters-docker-utilities.sln
+
+    - name: Test
+      shell: bash
+      run: dotnet dotcover test --dcReportType=HTML --nologo --no-build basisregisters-docker-utilities.sln
```
Comment (jvandaal) [inline .github/workflows/build.yml:91]: There are no tests (only samples), so don't know if we should keep this step?

Comment (lvermeulen) [reply .github/workflows/build.yml:91]: no, you can remove that step

Comment (lvermeulen) [reply .github/workflows/build.yml:91]: i will also take care of sonar

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: 2
+
+updates:  
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: daily
```
Comment (lvermeulen) [inline .github/dependabot.yml:7]: all intervals should be set to weekly

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
```
Comment (lvermeulen) [inline .github/dependabot.yml:12]: see above

**Review by lvermeulen — APPROVED** [review]

**Pull request merged**